src/components/UI/app_container.js

```diff
@@ -0,0 +1,153 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+const Wrapper = styled.div(props => ({
+  width: 'calc(25% - 15px)',
+  flexShrink: '0',
+  margin: '0 20px 20px 0',
+  '&:nth-of-type(4n + 0)': {
+    marginRight: '0px',
+  },
+
+  [`@media only screen and (max-width: ${props.theme.breakpoints.large})`]: {
+    width: 'calc(33.333% - 13.333px)',
+    // we are unsetting the 4n + 0 rule one above
+    '&:nth-of-type(4n + 0)': {
+      marginRight: '20px',
+    },
+    '&:nth-of-type(3n + 0)': {
+      marginRight: '0px',
+    },
+  },
+
+  [`@media only screen and (max-width: ${props.theme.breakpoints.small})`]: {
+    width: '100%',
+    marginRight: '0px',
+  },
+}));
+
+const StyledLink = styled(Link)({
+  display: 'block',
+  border: '1px solid #2a5974',
+  width: '100%',
+  borderRadius: '4px',
+  height: '158px',
+  backgroundColor: '#2f556a',
+  position: 'relative',
+  cursor: 'pointer',
+  overflow: 'hidden',
+  '&:hover': {
+    zIndex: 10,
+    boxShadow: '0px 0px 5px #549ac3',
+    // [`${AppIcon}`]: {
```

Comment:
I guess the latest changes aren't in yet?

Reply:
I couldn't find the way, maybe I should write a todo and an issue?

Reply:
If this way is like a workaround, and it works, not sure if we need a todo.

Maybe we'll figure out the composed form later, but if the PR works as intended, then its ok to leave out the comment and todo.